tests/jballerina-unit-test/src/test/resources/test-src/execution/service_decl_listener_return_error_project/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Do we need this file?

Reply:
We seems to be committing them
https://github.com/ballerina-platform/ballerina-lang/tree/master/tests/jballerina-unit-test/src/test/resources/test-src/execution/proj2

https://github.com/ballerina-platform/ballerina-lang/tree/master/tests/jballerina-unit-test/src/test/resources/test-src/execution/module_start_invocation_project

Reply:
ACK.